GettingStarted.md

```diff
@@ -0,0 +1,167 @@
+
+Now, probably the most important part of all is the *div* that belongs to the ```impress``` class. This *div* should contain all the HTML code you write, as everything outside that class will not be animated by impress.js. 
+
+Finally, we load the ```impress.js``` script from your local copy (if you have one) or from the cdn, if you do not have a local copy and execute 
+```
+impress().init()
+```
+to initialize impress.js. Now, let's continue on to explore more and more features of this amazing tool!
+
+## Creating slides
+Creating slides is fairly easy. You create a *div* that belongs to the class ```step``` and you are off to the races! Let me give you an example:
+```
+<div class="step">
+Hello World
+</div>
+```
+
+Now, if you reload the presentation, you start to see a \*slight\* problem. All your text is stacked... How do we work around that?
+
+Obviously, impress.js has an answer to it. You can add the following additional attributes to your div, to make it work:
+
+Attribute       | Explanation
+----------------|------------
+data-x          | Position the element along the x-axis (from left to right)
+data-y          | Position the element along the y-axis (from top to bottom)
+data-z          | Position the element along the z-axis (3D-Effect, move the element behind another one)
+data-rotate     | Rotate the element (if not using data-z!)
+data-rotate-x   | Rotate the element along the x-axis
+data-rotate-y   | Rotate the element along the y-axis
+data-rotate-z   | Rotate the element along the z-axis
+data-scale      | Scale the element. 
+
+These are the basic positioning options in impress.js. All of the attributes take Strings as arguments, so be aware of the fact, that you need to put quotation marks around the numbers! The *data-rotate* attributes take an angle in form of a String as argument.
+
+Now, that you have created the slides, you might want to style them. This is where CSS comes into play. Add another file to your project called, e.g., ```style.css```. 
+
+**NOTE:** As a general rule, you should avoid modifying the CSS properties of the built-in class *step*, as this will modify ***ALL*** the steps! You may change the font of it, but I recommend to instead add another class to that element and style that class. Whatever you do, do not mess with positioning and rotation of the div that belongs to the class *step*, but add a div inside of it, if you really have to mess with these properties. See the example below. Always position *steps* with the *data-* attribute!
+
```

Comment:
Sorry... I've thought about this some more, and I realize now this just isn't right.

We should think about this from the point of view that `impress-common.css` is *optional*. And if a user wouldn't use it, they rather MUST add CSS to the `.step` class. So the conclusion is that a user can and sometimes MUST add CSS to the `.step` class.

I would simply delete the first half of this section.But keep the latter half, from "Whatever you do..." to the end.